fluence-js-examples/browser-example/package.json

```diff
@@ -45,6 +46,7 @@
     ]
   },
   "devDependencies": {
+    "@fluencelabs/avm": "0.19.0-dynamic-import-wasm.5",
```

Comment:
We've just discussed a solution to avoid forcing users to depend on `avm` explicitly. Instead, `fluencelabs/fluence` will provide the script to copy both `avm.wasm` and `runner-script.js`